lib/constants.ex

```diff
@@ -117,4 +117,10 @@ defmodule Constants do
 
   @spec proposer_score_boost() :: non_neg_integer()
   def proposer_score_boost, do: 3
+
+  @spec attestation_subnet_count() :: non_neg_integer()
+  def attestation_subnet_count, do: 2 ** 6
```

Comment:
is this the same constant for every network?

Reply:
took it from
https://github.com/lambdaclass/lambda_ethereum_consensus/blob/main/docs/specs/p2p-interface.md#configuration

Reply:
Configuration -> changes depending on network